internal/model/label.go

```diff
@@ -7,3 +7,20 @@ type Label struct {
 	URL         string `json:"url"`
 	Color       string `json:"color"`
 }
+
+var AllLabels = []Label{
+	CommunityLabel,
+	BetaLabel,
+}
+
+var CommunityLabel Label = Label{
+	Name:        "Community",
+	Description: "This plugin is maintained by the Open Source Community.",
```

Comment:
@aaronrothschild Could you please confirm that this wording is fine

Reply:
Yes, this wording is good.